aea/aea.py

```diff
@@ -349,6 +349,8 @@ def teardown(self) -> None:
 
         :return: None
         """
+        logger.debug("[{}]: Calling teardown method...".format(self.name))
+        self.liveness.stop()
```

Comment:
`self.liveness.stop()` is already called in `stop` from which `teardown` is called. Please remove it here.

Reply:
I think about remove liveness object and use state of runtime.
i think runtime better represents the current state of the agent

Reply:
will be done in separate task